components/src/preact/wastewater/mutationsOverTime/wastewater-mutations-over-time.stories.tsx

```diff
@@ -65,3 +66,27 @@ export const Default: StoryObj<WastewaterMutationsOverTimeProps> = {
         },
     },
 };
+
+export const AminoAcids: StoryObj<WastewaterMutationsOverTimeProps> = {
+    ...Default,
+    args: {
+        ...Default.args,
+        sequenceType: 'amino acid',
+    },
+    play: async ({ canvas, step }) => {
+        await step('Wait for component to render', async () => {
+            await canvas.findByText('All segments');
+        });
+
+        await step("Click 'All segments' button", async () => {
+            canvas.getByRole('button', { name: 'All segments' }).click();
+            await expect(canvas.getByText('Select none')).toBeInTheDocument();
+            canvas.getByRole('button', { name: 'Select none' }).click();
+            await canvas.findAllByText('No data available for your filters.');
+            canvas.getByRole('checkbox', { name: 'S' }).click();
+            await canvas.findAllByText('S:Q493E');
+            const element = canvas.queryByText(/ORF1a:/);
+            await expect(element).toBeNull();
```

Comment:
Nitpick: 
```suggestion
            await expect(element).toBeInTheDocument();
```
is more expressive (even though technically it does the same)

Reply:
Hmm.. so here I don't want to see any ORF1a: mutations as I filtered just for S - so I think I need to keep it as is?

Reply:
Sorry, I meant: (missing a `not` in my other comment)
```suggestion
            await expect(element).not.toBeInTheDocument();
```